docs/user-guide/operations/deploy-on-kubernetes/deploy-greptimedb-cluster.md

```diff
@@ -1,44 +1,105 @@
+- [GreptimeDB Operator](./manage-greptimedb-operator/deploy-greptimedb-operator.md) (Assumes the local host has a matching installation of the GreptimeDB Operator) 
 
-### Create an etcd cluster
+## Create an etcd cluster (Optional) 
 
-First, establish an etcd cluster to support GreptimeDB by executing the following command:
+To install the ectd cluster, run the following `helm install` command. This command also creates a dedicated namespace `etcd-cluster` for the installation.
 
 ```shell
-helm upgrade \
-  --install etcd oci://registry-1.docker.io/bitnamicharts/etcd \
+helm install etcd oci://registry-1.docker.io/bitnamicharts/etcd \
   --set replicaCount=3 \
-  --set auth.rbac.create=false \
+  --set auth.rbac.create=false \****
   --set auth.rbac.token.enabled=false \
   --create-namespace \
   -n etcd-cluster
 ```
 
-After the installation,
-you can get the etcd cluster endpoints `etcd.etcd.svc.cluster.local:2379` from the installation logs.
-The endpoints are required for deploying the GreptimeDB cluster in the subsequent step.
+### Verify the etcd cluster installation
+Once installed, verify the status of the etcd cluster by listing the pods in the `etcd-cluster` namespace:
 
-### Create a GreptimeDB cluster
+```bash
+kubectl get pods -n etcd-cluster
+```
+
+You should see output similar to this:
+```bash
+NAME     READY   STATUS    RESTARTS     AGE
+etcd-0   1/1     Running   0            80s
+etcd-1   1/1     Running   0            80s
+etcd-2   1/1     Running   0            80s
+```
+
+## Deploy a minimum GreptimeDB cluster
 
-Deploy the GreptimeDB cluster, ensuring it connects to the previously established etcd cluster:
-  
+Next, deploy a minimum GreptimeDB cluster. The deployment will depend on an etcd cluster for coordination. If you already have one running, you can use its endpoint. If you followed the steps above, use `etcd.etcd-cluster.svc.cluster.local:2379` as the etcd endpoint.
+
+Run this command to install the GreptimeDB cluster:
 ```shell
 helm install greptimedb greptime/greptimedb-cluster \
   --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
   --create-namespace \
   -n greptimedb-cluster
 ```
 
-### Setting Resource requests and limits
+### Verify the GreptimeDB cluster installation
+Check that all pods are running properly in the `greptimedb-cluster` namespace:
+
+```bash
+kubectl get pods -n greptimedb-cluster
+```
+
+You should see output similar to this:
+```bash
+NAME                                      READY   STATUS    RESTARTS   AGE
+my-greptimedb-datanode-0                  1/1     Running   0          30s
+my-greptimedb-datanode-2                  1/1     Running   0          30s
+my-greptimedb-datanode-1                  1/1     Running   0          30s
+my-greptimedb-frontend-7476dcf45f-tw7mx   1/1     Running   0          16s
+my-greptimedb-meta-689cb867cd-cwsr5       1/1     Running   0          31s
+```
+
+## Advanced Configuration Options
 
```

Comment:
Maybe a link to the configuration.md file, and explain the rules for transforming the toml options into helm command options?